intake/source/textfiles.py

```diff
@@ -109,6 +109,12 @@ def to_dask(self):
         return db.from_delayed([dfile(f, self.decoder, self._read)
                                 for f in self._files])
 
+    def to_ibis(self):
+        import ibis.pandas
+        df = self.read()
```

Comment:
This is a list of text strings, not a dataframe, so you probably can't ibis this

Reply:
Ah, good point, I'll remove this.